MAUI/Carousel-View/Populating-Data.md

```diff
@@ -0,0 +1,275 @@
+
+{% highlight C# %}
+
+	carousel.BindingContext = new CarouselViewModel();
+
+{% endhighlight %}
+
+## Through Carousel Item
+
+Different set of views can be provided to every items through `ItemContent` property available in `SfCarouselItem` class.
+
+{% highlight C# %}
+
+using Syncfusion.Maui.Carousel;
+using System.Collections.ObjectModel;
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfCarousel carousel = new SfCarousel()
+            {
+                ItemWidth = 170,
+                ItemHeight = 250
+            };
+
+            ObservableCollection<SfCarouselItem> carouselItems = new ObservableCollection<SfCarouselItem>();
+
+            carouselItems.Add(new SfCarouselItem() { ItemContent = new Image() { Source = "carousel_person1.png", Aspect = Aspect.Fill } });
+            carouselItems.Add(new SfCarouselItem() { ItemContent = new Image() { Source = "carousel_person2.png", Aspect = Aspect.Fill } });
+            carouselItems.Add(new SfCarouselItem() { ItemContent = new Image() { Source = "carousel_person3.png", Aspect = Aspect.Fill } });
+            carouselItems.Add(new SfCarouselItem() { ItemContent = new Image() { Source = "carousel_person4.png", Aspect = Aspect.Fill } });
+            carouselItems.Add(new SfCarouselItem() { ItemContent = new Image() { Source = "carousel_person5.png", Aspect = Aspect.Fill } });
+
+            carousel.ItemsSource = carouselItems;
+
+            this.Content = carousel;
+        }
+    }
+}
+
+{% endhighlight %}
+
+and also carousel provides a support to display only the Image data with `Image` property in `SfCarouselItem` class.
+
+{% highlight C# %}
+
+using Syncfusion.Maui.Carousel;
+using System.Collections.ObjectModel;
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfCarousel carousel = new SfCarousel()
+            {
+                ItemWidth = 170,
+                ItemHeight = 250
+            };
+
+            ObservableCollection<SfCarouselItem> carouselItems = new ObservableCollection<SfCarouselItem>();
+            carouselItems.Add(new SfCarouselItem() { ImageName = "carousel_person1.png" });
+            carouselItems.Add(new SfCarouselItem() { ImageName = "carousel_person2.png" });
+            carouselItems.Add(new SfCarouselItem() { ImageName = "carousel_person3.png" });
+            carouselItems.Add(new SfCarouselItem() { ImageName = "carousel_person4.png" });
+            carouselItems.Add(new SfCarouselItem() { ImageName = "carousel_person5.png" });
+
+            carousel.ItemsSource = carouselItems;
+
+            this.Content = carousel;
+        }
+    }
+}
+
+{% endhighlight %}
+
+
+Similar way every item can be created and customized in case of different carousel item view is needed.
```

Comment:
Similarly, every item can be created and customized in case a different carousel item view is needed.